jar-service/src/main/java/com/sixsq/slipstream/run/RuntimeParameterResource.java

```diff
@@ -145,6 +145,9 @@ public void update(Representation entity) throws ResourceException,
 				abortOrReset(value, em);
 				setValue(value);
 			}
+		} else if (isGlobalStateParameter()) {
+			States newState = attemptChangeGlobalState(value);
+			setValue(newState.toString());
```

Comment:
With this change, we cannot force the State anymore.
The ability to force the state already helped us in the past to unblock some particular Run for some customers.
So I would prefer if we can keep this ability. Maybe by adding the following code between the line 190 and 191:
```java
} else if (getUser().isSuper()) {
    newState = toNewState;
```